TM1py/Services/CellService.py

```diff
@@ -477,25 +537,15 @@ def extract_cellset_raw(
         elif 'Ordinal' not in cell_properties:
             cell_properties.append('Ordinal')
 
-        if not elem_properties:
```

Comment:
Unfortunately we can't remove this code. Still needed to cater for non calculated members.